test/scraped/json_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+require 'pry'
```

Comment:
Probably best to put this line into `test_helper.rb`. (Not sure why it wasn't there already, sorry!)